AnnotationModule.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from BaseImage import printMaskHelper
+from skimage import io, img_as_ubyte
+import os
+
+import xml.etree.ElementTree as ET
+import numpy as np
+import cv2
+
+def get_points(xml_fname):
+    """Parses the xml file to get those annotations as lists of verticies"""
+    # create element tree object
+    tree = ET.parse(xml_fname)
+
+    # get root element
+    root = tree.getroot()
+
+    points = []
+
+    for annotation in root.findall('Annotation'):
+        for regions in annotation.findall('Regions'):
+            for region in regions.findall('Region'):
+                for vertices in region.findall('Vertices'):
+                    points.append([None] * len(vertices.findall('Vertex')))

```

Comment:
seems like this section should be done with list compression, similar to lines 33 below